tools/crm_simulate.c

```diff
@@ -473,12 +478,15 @@ setup_input(const char *input, const char *output)
     cib_object = NULL;
 
     if (rc < 0) {
-        fprintf(stderr, "Could not create '%s': %s\n",
-                output, pcmk_strerror(rc));
-        crm_exit(CRM_EX_CANTCREAT);
+        rc = pcmk_legacy2rc(rc);
```

Comment:
Also not needed?

Reply:
This was originally `rc = abs(rc)`, which I think I might still actually want it to be?  pcmk_rc_str will return "Unknown error" on a negative.

Reply:
Ah I missed that rc had been reused for write_xml_file(). I think the pcmk_legacy2rc() should go there, just after setting it, then this would be rc != pcmk_rc_ok.

Reply:
write_xml_file returns the number of bytes written or a negative error code so we pretty much need to leave it structured like it is.  I could make this line `rc = pcmk_legacy2rc(abs(rc))` though.

Reply:
Ah I get it now, it's fine as it is.

pcmk_legacy2rc() can handle -errno's